class/blue-app.ts

```
@@ -379,91 +379,104 @@ export class BlueApp {
+              unserializedWallet.init();
+              break;
             }
-            unserializedWallet.init();
-            break;
+            case 'lightningLdk':
+              // since ldk wallets are deprecated and removed, we need to handle a case when such wallet still exists in storage
+
+              unserializedWallet = new HDSegwitBech32Wallet();
+              unserializedWallet.setSecret(tempObj.secret.replace('ldk://', ''));
+              break;
+            case LegacyWallet.type:
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)